system/jlib/jtrace.cpp

```diff
@@ -71,6 +71,63 @@ namespace opentel_trace = opentelemetry::trace;
 
 using namespace ln_uid;
 
+/*
```

Comment:
Considered creating a custom sampler, but it's expected to perform a decision upfront, and doesn't not work well for our conditional span concept.